app/src/main/java/com/kabouzeid/gramophone/ui/activities/AlbumDetailActivity.java

```diff
@@ -327,6 +388,7 @@ public void setStatusbarColor(int color) {
     private void setAlbum(Album album) {
         this.album = album;
         loadAlbumCover();
+        loadWiki();
```

Comment:
I copied the code from biography loading. If we make it loading when the user opens the wiki I think It'll be slower.